pkg/services/multildap/testing.go

```diff
@@ -91,10 +91,10 @@ func (mock *MockMultiLDAP) Users(logins []string) (
 
 // User test fn
 func (mock *MockMultiLDAP) User(login string) (
-	*models.ExternalUserInfo, error,
+	*models.ExternalUserInfo, *ldap.ServerConfig, error,
```

Comment:
I ended up doing [shotgun surgery](https://refactoring.guru/smells/shotgun-surgery) by having to change three different mocks in three different places. Generally, it is a strong indication that something is wrong with the current state of these LDAP related tests but I decided to leave that for later (if that) - I will probably come back to this sometime this week and if I choose not to touch it we can decide whenever we can live with it or should we create an issue for it.